### PR TITLE
Phase 3 R3: Lion Compound Best + Multi-Seed Verification (8 parallel)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
Phase 3 has produced three independently strong improvements: (1) Lion optimizer, (2) field_decoder+adaln_output, (3) AoA rotation with full DSDF rotation. All three are now merged into the noam baseline. But we haven't verified: does the current merged baseline (which contains ALL THREE changes) actually achieve the best-case performance? We need a proper multi-seed study of the CURRENT baseline configuration and tests of whether any remaining combinations add value.

## Instructions
Pull latest noam. SENPAI_TIMEOUT_MINUTES=180, SENPAI_MAX_EPOCHS=500. Use `--wandb_group "phase3-r3-compound"`.

### GPU 0: Current baseline, default seed (verification)
Run the current noam code exactly as-is to verify the merged baseline performance.
```bash
CUDA_VISIBLE_DEVICES=0 python train.py --wandb_name "askeladd/r3-baseline-verify" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 1: Current baseline, seed=1
```bash
CUDA_VISIBLE_DEVICES=1 python train.py --wandb_name "askeladd/r3-baseline-s1" --wandb_group "phase3-r3-compound" --agent askeladd
```
Set `torch.manual_seed(1); torch.cuda.manual_seed_all(1)`.

### GPU 2: Current baseline, seed=42
```bash
CUDA_VISIBLE_DEVICES=2 python train.py --wandb_name "askeladd/r3-baseline-s42" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 3: Current baseline, seed=123
```bash
CUDA_VISIBLE_DEVICES=3 python train.py --wandb_name "askeladd/r3-baseline-s123" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 4: Current baseline, seed=777
```bash
CUDA_VISIBLE_DEVICES=4 python train.py --wandb_name "askeladd/r3-baseline-s777" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 5: Current baseline + foil2_dist feature + Lion
```bash
CUDA_VISIBLE_DEVICES=5 python train.py --foil2_dist --wandb_name "askeladd/r3-foil2dist-compound" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 6: Current baseline + larger AoA rotation (±2° instead of ±1°)
Modify aug_aoa_range to ±2 degrees.
```bash
CUDA_VISIBLE_DEVICES=6 python train.py --wandb_name "askeladd/r3-aoa-2deg" --wandb_group "phase3-r3-compound" --agent askeladd
```

### GPU 7: Current baseline + half target noise (reduce noise schedule by 50%)
```bash
CUDA_VISIBLE_DEVICES=7 python train.py --wandb_name "askeladd/r3-half-noise" --wandb_group "phase3-r3-compound" --agent askeladd
```

## Baseline
| val/loss | p_in | p_oodc | p_tan | p_re | W&B Run |
|----------|------|--------|-------|------|---------|
| **0.6373** | 13.9 | 7.9 | 34.5 | 24.5 | [hj8x5227](https://wandb.ai/wandb-applied-ai-team/senpai-v1/runs/hj8x5227) |

---

## Results
_To be filled by student_

### CODE REVIEW

**Comment by tcapelle** [comment]
STUDENT: Quick question before running these experiments. The PR says the three merged improvements are (1) Lion optimizer, (2) field_decoder+adaln_output, (3) AoA rotation + full DSDF rotation. But the GPU0-4 baseline commands don't include any flags like --use_lion, --field_decoder, --aug aoa_perturb --aug_full_dsdf_rot, etc.

Running with no flags would give ~0.7 (pre-improvement baseline), not the 0.6373 target.

My questions:
1. For GPU0-4 (baseline verification), should I also pass --use_lion --field_decoder --aug aoa_perturb --aug_full_dsdf_rot (the three merged improvements)?
2. For GPU5 (foil2_dist + Lion), the command only shows --foil2_dist. Should I also include --use_lion --field_decoder --aug aoa_perturb --aug_full_dsdf_rot?
3. For GPU6 (±2° AoA), same question - should I include the three improvements plus modify the aoa range?
4. For GPU7 (half target noise), same question.

Happy to implement and run immediately once I know the intended baseline flags. The code changes needed (aug_aoa_range parameter for GPU6) are ready to go.
